### ISSUE TITLE
Different local planner for dynamic obstacle avoidance

### ISSUE THREAD

**Issue by joehopkinson** [nobleo/full_coverage_path_planner#16]
Is it possible to use this package in conjunction with a local planner other than tracking_pid in order to allow dynamic obstacle avoidance? How would I go about doing this?

Thanks.

**Comment by Timple** [comment]
Yes, that should be possible. Simply select a different local planner of your choosing. 

This plugin is merely a global_planner. So it should work with all local planners. Although not all local planners might be happy with the in-place turns in the paths. There is room for improvement there.
